pod/perlfunc.pod

```diff
@@ -4602,11 +4603,16 @@ opens a filehandle to a newly created empty anonymous temporary file.
 sensible mode to use.)  You will need to
 L<C<seek>|/seek FILEHANDLE,POSITION,WHENCE> to do the reading.
 
-Perl is built using PerlIO by default.  Unless you've
-changed this (such as building Perl with C<Configure -Uuseperlio>), you can
-open filehandles directly to Perl scalars via:
 
-    open(my $fh, ">", \$variable) || ..
+=item Opening a filehandle into an in-memory scalar
+
+You can open filehandles directly to Perl scalars instead of a file or
+other resource external to the program. Accomplish this by providing a
+reference to that scalar as the third argument to C<open>, like so:
+
+ open(my $memory, ">", \$var)
+     or die "Can't open memory file: $!";
+ print $memory "foo!\n";              # output will appear in $var
```

Comment:
Might be nice to specifically point out in this example that only bytes should be printed just like with regular files.

Reply:
Not so sure, since this contrived example is explicitly only storing and printing the literal `foo!\n`, so dodges any wide-character issues (if that's what you mean). That sort of admonition seems more in the jurisdiction of `print`'s documentation, right?

Reply:
It's more the jurisdiction of PerlIO::scalar which is fine, just a thought.